File: JS/exemplo3.js

function genericFunction(number){
  var count
  for(let i = number/2; i <= number; i++){
    for(let j = 1; j + number/2 <= number; j++){
      for(let k = 1; k <= number; k = k*2){
        count++
      }
    }
  }
  return 1
}

File: exemplo3.c
#include <stdio.h>

void exemplo3(int number){
  int i,j,k,count;
  for(i = number/2; i <= number; i++)
    for(j = 1; j + number/2 <= number; j++)
      for(k = 1; k <= number; k = k*2)
        count = count + 1;
  return;
}

